packages/web/src/pages/upload-page/UploadPage.tsx

```diff
@@ -3,14 +3,14 @@ import { FeatureFlags } from '@audius/common'
 import { useFlag } from 'hooks/useRemoteConfig'
 
 import UploadPageLegacy from './UploadPageLegacy'
-import UploadPageNew from './UploadPageNew'
+import { UploadPageNew } from './UploadPageNew'
 
 export const UploadPage = (props: any) => {
   const { isEnabled: isRedesignEnabled } = useFlag(
     FeatureFlags.UPLOAD_REDESIGN_ENABLED
   )
 
-  return isRedesignEnabled ? (
+  return !isRedesignEnabled ? (
```

Comment:
I assume this was to enable your local testing? Should remove it, right?

Reply:
yes great catch